04-workshop_ggplot2.Rmd

```diff
@@ -278,26 +278,26 @@ There are choices about the chart to use but also the details of the chart
 
 class: center, middle
 
-# Choices
+# Choices  
 
-</br> 
+ 
 
-</br> 1. What shape will represent the data points?
-</br> 
-</br> 
+ 1. What shape will represent the data points?  
```

Comment:
Unfortunately </br> is needed for this bullet as otherwise the 1. is at the far left and the text is centred.